### PR TITLE
Add info to checks

### PR DESCRIPTION
**Pull request by grstk**
Enables users to add more information about project checks. That info is included in assignment notification sent to checker.

### CODE REVIEW

File: app/controllers/project_checks_controller.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ class ProjectChecksController < ApplicationController
   expose(:check) do
     project_checks_repository.find(
       params[:id] || params[:project_check_id],
-    )
+    ).decorate
```
Comment (lubieniebieski) [inline app/controllers/project_checks_controller.rb]: IMO decorated `check` should be used in `show` template only - this way we're using an additional object/methods everywhere (even if we don't need them)

File: app/views/project_checks/edit.slim [hunk 2]
```diff
@@ -0,0 +1,6 @@
+h1 #{check.review} info for project #{check.project_name}
+
+== render 'form'
```
Comment (lubieniebieski) [inline app/views/project_checks/edit.slim]: double `==` ?

Comment (grstk) [inline app/views/project_checks/edit.slim]: Hm, I've read somewhere that this is how prerendered html should be included in slim, though now I see that single `=` works just as well.

File: app/controllers/project_checks_controller.rb (hunk 1 above)
Comment (grstk) [inline app/controllers/project_checks_controller.rb]: ok, will move decoration to template

**Pull request merged**